src/olympia/reviewers/views.py

```diff
@@ -388,10 +389,10 @@ def save_motd(request):
         raise PermissionDenied
     form = forms.MOTDForm(request.POST)
     if form.is_valid():
-        set_config('editors_review_motd', form.cleaned_data['motd'])
-        return redirect(reverse('editors.motd'))
+        set_config('reviewers_review_motd', form.cleaned_data['motd'])
```

Comment:
Needs a migration to change the config name.  Or a manual migration of poking Jorge to set it again after deploy 😜

Reply:
I was thinking of doing this the manual way :p